package br.co.progresso.concurso.exception;

public class ConcursoNaoEncontradoException extends RuntimeException {
    public ConcursoNaoEncontradoException(Long id) {
        super("Concurso n√£o encontrado com o id: " + id);
    }
}